forge-gui/res/cardsfolder/t/the_matrix_of_time.txt

```diff
@@ -0,0 +1,14 @@
+Name:The Matrix of Time
+ManaCost:no cost
+Types:Plane Gallifrey
+T:Mode$ PlaneswalkedTo | ValidCard$ Plane.Self | Execute$ TrigExile | TriggerDescription$ When you planeswalk to CARDNAME, each player exiles the top card of their library.
+SVar:TrigExile:DB$ Dig | DigNum$ 1 | ChangeNum$ All | Defined$ Player | DestinationZone$ Exile
+S:Mode$ Continuous | Affected$ Card.ExiledWithSource | MayPlay$ True | Condition$ PlayerTurn | EffectZone$ Command | AffectedZone$ Exile | Description$ During your turn, you may play lands and cast spells from among cards exiled with CARDNAME.
+T:Mode$ SpellCast | ValidCard$ Card | ValidSA$ Spell.MayPlaySource | TriggerZones$ Command | Execute$ TrigLoseLife | TriggerDescription$ Whenever you play a land or cast a spell from among cards exiled with CARDNAME, that card's owner loses 3 life and exiles the top card of their library.
```

Comment:
the ValidSA are incorrect because they're not connected like that ("this way")
which means you could also get it to trigger by casting exiled _Squee, the Immortal_ with its intrinsic ability